virttest/utils_test/libvirt.py

```diff
@@ -3442,8 +3442,11 @@ def modify_vm_iface(vm_name, oper, iface_dict, index=0):
     :param iface_dict: The dict restore need updated items like iface_driver,
                        driver_host, driver_guest and so on
     :param index: interface index in xml
+    :param virsh_instance: virsh instance object. If you want to modify remote
```

Comment:
Better not use 'I ' ,'we', 'you' in param description

Reply:
Okay, removed.